materials/dplyr.md

```diff
@@ -5,79 +5,102 @@ title: Working with Tabular Data
 language: R
 --- 
 
-## Packages
+> Remember to
+> 
+> * download [`surveys.csv`](https://ndownloader.figshare.com/files/2292172).   
+> * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171).
+> * `setwd()` in R.
```

Comment:
It's a reminder to the instructor to set the working directory in their R console before the start of class to be able to access `surveys.csv` without a path, such as `read.csv("surveys.csv")`.